packages/spacecat-shared-data-access/test/unit/models/api-key.test.js

```diff
@@ -54,9 +54,9 @@ describe('ApiKey Model tests', () => {
       expect(apiKey.getCreatedAt()).is.not.empty;
     });
 
-    it('creates an ApiKey object for a user with scope - imports.write_all_domains', () => {
-      const apiKey = createApiKey({ ...validApiKey, scopes: [{ name: 'imports.write_all_domains', domains: [] }] });
-      expect(apiKey.getScopes()).to.deep.equal([{ name: 'imports.write_all_domains', domains: [] }]);
+    it('creates an ApiKey object for a user with scope - imports.all_domains', () => {
+      const apiKey = createApiKey({ ...validApiKey, scopes: [{ name: 'imports.all_domains', domains: [] }] });
```

Comment:
To be valid, this key should have the `imports.write` scope as well, right?

Reply:
Yes. Should that check be here?
Or in spacecat-api-service?

Reply:
Added the additional scope to the test.